scripts/deploy_scale_dashboard.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+import json
+import os
+
+import boto3
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss.util.aws import ARN
+from dss.stepfunctions import step_functions_arn
+
+stage = os.environ.get('DSS_DEPLOYMENT_STAGE')
+dev_stage = 'dev'
+region = ARN.get_region()
+accountid = ARN.get_account_id()
+
+checkout_bundle_arn_prefix = f"arn:aws:lambda:{region}:{accountid}:function:dss-scalability-test-{stage}:domovoi-stepfunctions-task-CheckoutBundle"
+upload_bundle_arn_prefix = f"arn:aws:lambda:{region}:{accountid}:function:dss-scalability-test-{stage}:domovoi-stepfunctions-task-UploadBundle"
+download_bundle_arn_prefix = f"arn:aws:lambda:{region}:{accountid}:function:dss-scalability-test-{stage}:domovoi-stepfunctions-task-DownloadBundle"
+dss_s3_copy_arn = f"arn:aws:states:{region}:{accountid}:stateMachine:dss-s3-copy-sfn-{dev_stage}"
+
+
+def get_metrics_array(arn_template, cnt):
+    metrics_array = [["AWS/States", "LambdaFunctionRunTime", "LambdaFunctionArn", arn_template + '{}'.format(0),
+                      {"yAxis": "left", "period": 10}]]
+    for idx in range(1, cnt):
+        metrics_array.append(
+            ["...",
+             arn_template + '{}'.format(idx),
+             {"period": 10}])
+    return metrics_array
+
+
+dashboard_def = {
+    "widgets": [
+        {
+            "type": "metric",
+            "x": 0,
+            "y": 24,
+            "width": 12,
+            "height": 6,
+            "properties": {
+                "view": "timeSeries",
+                "stacked": True,
+                "metrics": [
+                    ["AWS/ES", "CPUUtilization", "DomainName", f"dss-index-{stage}", "ClientId", "861229788715",
+                     {"period": 10}],
+                    [".", "WriteThroughput", ".", ".", ".", ".", {"period": 10}],
+                    [".", "WriteIOPS", ".", ".", ".", ".", {"period": 10}]
+                ],
+                "region": region,
+                "period": 300,
+                "title": "Elastic search"
+            }
+        }
+        ,
```

Comment:
please remove extra newline here and in the rest of the file

Reply:
fixed